### PR TITLE
Finalize content for about page

### PR DESCRIPTION
**Pull request by santina**
## What

About page was lacking in content for qualifications of our volunteers (companies worked/working at, degrees, and skills). This PR add them, and also did some formatting.

## Why Do

Fit the design and add the content to about page

## How Do
Some modifications beyond what's obvious from the design:
- a mix of MUI icons and svg exported from Figma.
- slight modification to the values cards so they appear to have the same size - I also found them to be less verbose this way.
- rearrange the qualification item list so that the ones that are similar size appear next to each other
- from the above two points, I did that so I didn't have to fix the sizes of the list item or cards, which would require modifying the underlying components

## Show Me

localhost_3000_about_new_small
localhost_3000_about_new

### CODE REVIEW

**Comment by santina** [comment]
preview here https://open-seattle-website-git-santina-aboutcontent-dev-openseattle.vercel.app/about_new

File: pages/about_new.tsx [hunk 1]
```diff
@@ -4,109 +4,150 @@ import {
...
+      Open Seattle partners with other nonprofits in order to amplifying their
+      impact, and to uplift communities with technology.
+    </Typography>
+  </AboutUsSection>
+)
+
+const OurValueSection = ({ theme }) => (
+  <AboutUsSection backgroundColor={theme.palette.background.default}>
     <Typography variant="headlineMedium">Our values</Typography>
     <CardRowContainer>
```
Comment (mistyb01) [inline pages/about_new.tsx:152]: I think wrapping the cards with a Box with grid properties, isntead of CardRowContainer, would remove that weird wrapping behavior when resizing the screen! You can use this:
```
          <Box
            sx={{
              display: 'grid',
              gridTemplateColumns: { xs: '1fr', lg: 'repeat(3, 1fr)' },
              gap: '2rem',
            }}
          >
```

File: pages/about_new.tsx [hunk 2]
```diff
@@ -4,109 +4,150 @@ import {
...
-    maxWidth={false}
-  >
-    <Box sx={{ gap: '2rem', display: 'grid', margin: '2rem' }}>
-      <Typography variant="headlineMedium">What we do</Typography>
-      <Typography variant="bodyLarge">
-        Open Seattle partners with other nonprofits in order to amplifying their
-        impact, and to uplift communities with technology.
-      </Typography>
+const AboutUsHeroSection = ({ theme, extraSmallScreen }) => (
+  <SectionContainer backgroundColor={theme.palette.primary.main}>
```
Comment (mistyb01) [inline pages/about_new.tsx:90]: Minor thing but from the figma tablet design, it looks like the hero width should match the width of the sections below it. 
I also wonder if making this hero into a component would be useful for keeping things consistent (or if it's overkill lol)

**Review by mistyb01 — APPROVED** [review]
Thanks Santina, this looks great! 👍

**Comment by santina** [comment]
thank you @mistyb01 for reviewing!!

**Pull request merged**